exporter/jaegerexporter/exporter_test.go

```diff
@@ -192,10 +192,10 @@ func TestMutualTLS(t *testing.T) {
 	// start gRPC Jaeger server
 	tlsCfgOpts := configtls.TLSServerSetting{
 		TLSSetting: configtls.TLSSetting{
-			CAFile:   caPath,
 			CertFile: serverCertPath,
 			KeyFile:  serverKeyPath,
 		},
+		ClientCAFile: caPath,
```

Comment:
to force mTLS client CA should be used instead of CA https://smallstep.com/hello-mtls/doc/server/go